modules/api-gateway-domains-and-certs/main.tf

```diff
@@ -1,6 +1,7 @@
 
 module "primary_region" {
-  source = "./modules/certificate-and-domain"
+  source  = "sil-org/serverless-api-dns-for-failover/aws//modules/api-gateway-domains-and-certs/modules/certificate-and-domain"
+  version = "~> 0.6.0"
```

Comment:
Why was this changed? (Since it's a local/internal module, it will automatically have the same version as the code using it, so I don't think we need to add a version constraint, or modify this line at all, for that matter, though I might be missing something.)